broker/lib/cdn.py

```diff
@@ -1,8 +1,14 @@
 from broker.models import ServiceInstanceTypes
 
 
-def is_cdn_instance(service_instance):
+def is_cdn_instance(service_instance) -> bool:
     return service_instance.instance_type in [
         ServiceInstanceTypes.CDN.value,
         ServiceInstanceTypes.CDN_DEDICATED_WAF.value,
     ]
```

Comment:
maybe late to be checking this, but why aren't we just using `isinstance(service_intance, CDNServiceInstance)`

Reply:
Absolutely no reason. Is there an inherent benefit to one approach vs the other?

Reply:
`isinstance` is a builtin, so I think it's a bit more idiomatic, and probably faster (although speed isn't really critical here - nobody would notice if this took seconds to perform, except during testing)